Runtime/Components/ConeRayAnglesEstimator/IConeRaySegmentComputation.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+using ubco.ovilab.HPUI.Interaction;
+
+namespace ubco.ovilab.HPUI
+{
+    public interface IConeRaySegmentComputation
+    {
+        /// <summary>
+        /// For a given segment, computes the List of <see cref="HPUIInteractorRayAngle">.
+        /// </summary>
+        /// <param name="segment">
+        ///   The <see cref="HPUIInteractorConeRayAngleSegment"/> for which cone
+        ///   angles are being computed.
+        /// </param>
+        List<HPUIInteractorRayAngle> EstimateConeAnglesForSegment(HPUIInteractorConeRayAngleSegment segment, IEnumerable<ConeRayComputationDataRecord> interactionRecords);
+    }
+
+    /// <summary>
+    /// Segments of the cone estimation. Corresponds to the fields of <see cref="HPUIInteractorConeRayAngles"/>
+    /// </summary>
+    public enum HPUIInteractorConeRayAngleSegment
+    {
+        IndexDistalSegment,
+        IndexIntermediateSegment,
+        IndexProximalSegment,
+        MiddleDistalSegment,
+        MiddleIntermediateSegment,
+        MiddleProximalSegment,
+        RingDistalSegment,
+        RingIntermediateSegment,
+        RingProximalSegment,
+        LittleDistalSegment,
+        LittleIntermediateSegment,
+        LittleProximalSegment
```

Comment:
Can we not just reuse the enums from XRI?

Reply:
Also please assign the enums an explicit integer value

IndexDistalSegment = 0,
IndexIntermediateSegment = 1

etc

It will prevent from breaking serialization in editor if any changes are made to this enum (i.e in case i wanna add radial in the future?)

Reply:
> Can we not just reuse the enums from XRI?

That has phalanges we don't need. Plus, as you point out, we may want to expand it in different directions.

> Also please assign the enums an explicit integer value
> It will prevent from breaking serialization

Good point.